cms/toolbar/items.py

```diff
@@ -222,11 +224,16 @@ def add_break(self, identifier=None, position=None):
         return item
 
     def get_items(self):
-        return self.items
```

Comment:
why is it necessary to set this on the item instances? don't they  already have they're own disabled state?